### DISCUSSION TITLE
UMinho – thesis rejected due to font size

### DISCUSSION THREAD

**Discussion by dlrsb** [joaomlourenco/novathesis#284] [Q&A] [answered]
Hi,

Thank you @joaomlourenco for creating this template!

I recently submitted my PhD thesis, which I had typset using the Novathesis template (version 6.4.7), and UMinho did not accept it due to the following: **“Devolução do processo pois não está conforme o Despacho RT-31/2019. Ponto 16. ".... fonte NewsGotT e uma dimensão de 12 para a letra de texto ..." - a dimensão da tese está com a dimensão de 11,96.”**

It seems that [the definition of 1 pt in LaTeX is different from the standard used in Postscript/Word/etc](https://tex.stackexchange.com/questions/200934/why-does-a-tex-point-differ-from-a-desktop-publishing-point). LaTeX calls the Postscript standard “big point” (bp).

I tried to fix this by changing the memoir setup to 12 bp instead of 12 pt in the 0_memoir.tex file and also in the univ-defaults.ldf file, but this didn’t work and it resulted in a file with an even smaller font size (I’m assuming it used the default fontsize for memoir instead). Is there any way I could change the novathesis.cls to use bp instead of pt/to convert between the two?

Thanks.

**Comment by joaomlourenco** [comment]
@dlrsb I'm not sure if we should be happy or cry with your message.

Cry because 11.96 vs 12.00 is absolutely imperceptible by any terms except detailed analysis of the PDF internals, and they rejected the submission based on this claim.

But, on the on the hand, we may be really happy, as it seems that is the only issue remaining to be solved.  :)

I found the solution, but I'm not sure how to make it work for you, depends on which precise version (commit) you are using, but try adding one(-by-one) of the following to your "Config/5_packages.tex":

```latex
\AtBeginDocument{\@setfontsize\normalsize{12bp}{14.5bp}}
\AtBeginDocument{\makeatletter\@setfontsize\normalsize{12bp}{14.5bp}}
\AtBeginDocument{@setfontsize\normalsize{12bp}{14.5bp}\makeatother}
\AtBeginDocument{\makeatletter\@setfontsize\normalsize{12bp}{14.5bp}\makeatother}
```
I believe one of the above should work.  Please let me know that it really worked for you.

I have added the fix to the uminho configuration file.  Newer versions will be ok.  :)

**Reply by dlrsb** [reply]
Thank you for the quick reply! Unfortunately, I tested the commands one-by-one and none of them worked for the version I'm using (Version 2021-06-19 [6.4.7], I don't have information on the specific commit). I did modify the template slightly to eliminate the covers that were being generated and to include my own, but I wouldn't expect this to affect the behavior of the commands you suggested.

LaTeX now throws the following errors: 
You can't use `\spacefactor' in vertical mode. \begin{document}
Missing $ inserted. \begin{document}
Missing $ inserted. (on the line following \begin{document}

I also noticed these warnings: 
Command \normalsize invalid in math mode.
Some font shapes were not available, defaults substituted.

My thesis has two equation environments and a few inline equations, but I don't use the \normalsize command within math mode... I wonder if fontsize needs to be redefined specifically for math mode as well?

I can just try use the most recent version of the template instead and see if it works...

**Comment by joaomlourenco** [comment]
It works in the template.
It is not working if I apply the same patch to another thesis from uminho that I have access to.  I have to figure out what is failing. 
Maybe you can download the latest version of the template (which already has the patch) and try do re-apply your changes?  
Or please wait a couple of days until I devise a more "bullet proof" version of the fix.

BTW, why did you replace the covers?

![image](https://user-images.githubusercontent.com/2064643/151780529-b9a862b9-5c42-433d-80bd-bf78ab3a2ed6.png)

**Reply by dlrsb** [reply]
Ok, I'm going to try using the latest version and I'll let you know if it works.

I replaced the covers because the template version I had been using was affected by this issue https://github.com/joaomlourenco/novathesis/issues/254, which I only became aware of shortly before the deadline. Since I also needed to include funder logos, it was easier for me to just edit UMinho's official template for the covers & spine and comment out the lines in univ_defaults.ldf that generated the covers. If I remember correctly, this also fixed an issue with the page numbering, since the copyright page is supposed to be page ii and before my changes I think it was page iv.

**Reply by b-pereira** [reply]
You could use this https://github.com/b-pereira/UMINHO-PHD-MSc-Covers/releases. For the time being, the two main documents are standalone. The cover and back cover verses can be created with the given instructions. 

Given that you followed my instruction in #254 this is for a follow-up by other people. That nitpicking is not consistent within UMINHO Schools and that's appalling. There are a number of "issues" with the template if they want to commit to Adobe and text processors standards. For instance, measure the line space between the text baselines. I don't know if this template defined 1.5 or it is using \onehalfspacing - in LaTeX, for 12pt that factor is 1.241 and not 1.5 and relative to the font size and baseline [Why is the line spread factor as it is?](https://tex.stackexchange.com/questions/30073/why-is-the-linespread-factor-as-it-is).  This is sad because we use LaTeX because of LaTeX's typographical quality. Everything was measured within the packages and classes to be precise and have good typographical quality.

**Reply by dlrsb** [reply]
> It works in the template. It is not working if I apply the same patch to another thesis from uminho that I have access to. I have to figure out what is failing. Maybe you can download the latest version of the template (which already has the patch) and try do re-apply your changes? Or please wait a couple of days until I devise a more "bullet proof" version of the fix.

@joaomlourenco, what tool did you use to get the font measurements shown in the figure? I compiled the template (downloaded today, from master. uminho-defaults.ldf has the patch) without changing anything besides 1_novathesis.tex and asked someone who has access to Adobe Acrobat Professional to tell me the font size... and they said it's 11.96 pt. I can't understand why...

![imagem](https://user-images.githubusercontent.com/14823653/151851533-369eca69-c8a0-4e5b-949d-c7ffc480c620.png)

The only section where Adobe says the font is 12 pt is the Copyright section:

![imagem](https://user-images.githubusercontent.com/14823653/151869645-7669a035-d433-4b49-8824-639af2be43b8.png)

I extracted text from the PDF with the corresponding font sizes using the pdfminer package in Python and I get the same values - 12 pt in the Copyright text, and 11.95519999999999 pt elsewhere, including the "Lorem ipsum dolor" part

**Comment by b-pereira** [comment]
> Hi,
> 
> Thank you @joaomlourenco for creating this template!
> 
> I recently submitted my PhD thesis, which I had typset using the Novathesis template (version 6.4.7), and UMinho did not accept it due to the following: **“Devolução do processo pois não está conforme o Despacho RT-31/2019. Ponto 16. ".... fonte NewsGotT e uma dimensão de 12 para a letra de texto ..." - a dimensão da tese está com a dimensão de 11,96.”**
> 
> It seems that [the definition of 1 pt in LaTeX is different from the standard used in Postscript/Word/etc](https://tex.stackexchange.com/questions/200934/why-does-a-tex-point-differ-from-a-desktop-publishing-point). LaTeX calls the Postscript standard “big point” (bp).
> 
> I tried to fix this by changing the memoir setup to 12 bp instead of 12 pt in the 0_memoir.tex file and also in the univ-defaults.ldf file, but this didn’t work and it resulted in a file with an even smaller font size (I’m assuming it used the default fontsize for memoir instead). Is there any way I could change the novathesis.cls to use bp instead of pt/to convert between the two?
> 
> Thanks.

This is much unexpected. At least, one PhD thesis with the LaTeX 12pt font size was already accepted. What is your school?

**Reply by dlrsb** [reply]
Escola de Engenharia. Someone from my research group submitted his thesis a few months before I did and UMinho did not complain about this. And I've seen non-compliant theses in Repositorium (e.g. font size 11pt)...

**Reply by b-pereira** [reply]
There are School of Architecture theses and they look like magazines with full-page images without pagination, there others that use Arial instead of News Got T, and more. I wonder what gave it away ... if the font size is 11,96 maybe the line space gave it away because it is easier to measure. See the image below:

<img src="https://user-images.githubusercontent.com/944511/151809707-c31e1428-0059-4deb-9b0d-7ac1a9254822.png" width="300" />


So to be precise **the leading** is 12pt x 1.5 which is 18pt which in turn is 6.35mm. The leading with 11.96pt x 1.5  is 17,94pt which in turn is 6.3288mm. As @joaomlourenco image above shows, the leading is 14.4pt which indicates that the line height is 1.2 (see [how to calculate line-height from psd file? (leading+ font size)](https://stackoverflow.com/questions/17088344/how-to-calculate-line-height-from-psd-file-leading-font-size)). The 1.2 line-height didn't come from thin air: Word does some pe­cu­liar line-spac­ing math (see [Confusion: \onehalfspacing vs. \spacing vs. Word vs. the world?](https://tex.stackexchange.com/questions/65849/confusion-onehalfspacing-vs-spacing-vs-word-vs-the-world#65985))

> Microsoft Word doesn't do line spacing correctly (unless you multiply the pt value of the text yourself and use the "Exactly" line-spacing option). If you actually measure line height in a graphics-editing program—from the top of the tallest letter to the bottom of the lowest letter—\onehalfspacing uses exactly 0.5 of this height below each line.
 From http://practicaltypography.com/line-spacing.html,

> >    For most text, the op­ti­mal line spac­ing is be­tween 120% and 145% of the point size.

> In Microsoft Word,

>>    To get line spac­ing in the 120–145% range, use a Multiple value of 1.03–1.24. (Not 1.20–1.45—as noted above, Word uses pe­cu­liar line-spac­ing math.)

> So to answer your question, if you want true 1.5 line spacing, go with \onehalfspacing. However, to most people, 1.5 line spacing probably means using the (incorrect) "1.5 line spacing" option in Word. It would be best to ask your lecturer for an example paper.

This begs the question: given that all LaTeX typography uses non-Adobe point metrics, maybe all the elements aren't a complete match with the standards. Maybe I'm not all that familiar if both systems are separated. @joaomlourenco could answer that more specifically.

Peter Flynn paper ["Rolling your own Document Class: Using LATEX to keep away from the Dark Side"](https://github.com/b-pereira/UMINHO-PHD-MSc-Covers/blob/feature/resources/resources/Rolling%20your%20own%20Document%20Class:%0AUsing%20L%20A%20TEX%20to%20keep%20away%20from%20the%20Dark%20Side%0APeter%20Flynn.pdf) and YouTube lecture ["Rolling your own document class"](https://www.youtube.com/watch?v=anOUUr4_w_Y) gives some hints on how to change the font size without hammering with a quick fix. Flynn dissects the article class on how the font size works as well the paper sizes, etc. One of the links answers above shows how to do it ([Why is the line spread factor as it is?](https://tex.stackexchange.com/questions/30073/why-is-the-linespread-factor-as-it-is)). Font sizes are not calculated, they are concatenated. In the article class example the default 10pt and you choose 12pt, the 2 is concatenated to make 12pt (more or less like that) and always refer to a point definition in `@ptsize.clo`.

So, instead of using the memoir class defaults `clo` sizes, one could try to use them directly. The example below is for `size11.clo`. `@xipt` refers to 11 in roman numbering.  If all the point sizes are from `@ptsize.clo` maybe all that uses it, as the memoir class, will affect the whole document metrics. The code snippet below is from [size11.clo](https://www.tug.org/svn/texlive/trunk/Master/texmf-dist/tex/latex/base/size11.clo?view=co)

```latex
\@setfontsize\normalsize\@xipt{13.6}%
```

In short, maybe they measured the leading or as @joaomlourenco they look to the PDF internals. @joaomlourenco  what is the program that you use above to see the font size and leading? Also, @joaomlourenco could you comment on this tangent of mine?

**Reply by b-pereira** [reply]
> Escola de Engenharia. Someone from my research group submitted his thesis a few months before I did and UMinho did not complain about this. And I've seen non-compliant theses in Repositorium (e.g. font size 11pt)...

@dlrsb Biomedical Engineering?

**Reply by dlrsb** [reply]
@b-pereira Yes, I'm in the Biomedical Engineering program.

**Reply by b-pereira** [reply]
I helped your colleague via PM on Facebook. Your colleague used this template version https://github.com/b-pereira/novathesis. If you have already submitted your thesis this might be irrelevant, but if you still want help contact me via https://www.facebook.com/pereirabruno05/ via PM.

**Reply by b-pereira** [reply]
@dlrsb I checked my version with PDF X-Change Editor. You can too. I think that feature is free. https://www.tracker-software.com/product/pdf-xchange-editor

![imagem](https://user-images.githubusercontent.com/944511/151882262-f5fb6b71-c710-4578-b6fb-68e28d5468b9.png)

EDIT: I checked a two-month version of THIS template. 12pt in it also.

Used Libre Office Draw 

![imagem](https://user-images.githubusercontent.com/944511/151888561-49036b63-5459-4ae5-a141-fadc72b923f7.png)

12pt also

**Reply by b-pereira** [reply]
@dlrsb send me your version if you don't mind. I can't check if my PDF Editor is rounding up the units.

**Reply by dlrsb** [reply]
@b-pereira https://www.dropbox.com/s/liyjvdta9afuznv/template.pdf?dl=0. I think it's probably rounding. I also get 12 pt when checking with PDF X-Change Editor and LibreOffice Draw. But if UMinho uses something like Adobe Acrobat Professional they're probably going to see 11.96...

**Reply by b-pereira** [reply]
Yes, it is. Tough luck, someone had the Adobe Professional, Adobe InDesign or Adobe Illustrator installed to check that 0.4 pt difference in the Scientific Council. Look, I used at least 2 PDF Editors - Libre Office and PDF X-Change Editor - in both the font size was 12pt. However, in Inkscape the font size was 11.955pt.

#### PDF X-Change Editor
![imagem](https://user-images.githubusercontent.com/944511/151888862-83a7131f-2def-4a9d-b447-86cdfe697782.png)

#### Inkscape 

![imagem](https://user-images.githubusercontent.com/944511/151891561-951d354a-623f-4675-afb3-dee99435c1b4.png)

**Reply by dlrsb** [reply]
@b-pereira I confirmed that the copyright section is also 12 pt in Inkscape, unlike the rest of the text... so it seems to me that the patch works for that section and not the rest??

**Reply by b-pereira** [reply]
Em bom Português, o argumento do CC da EEUM não é contra o tamanho da fonte, mas um argumento contra o uso do LaTeX. A Escola de Engenharia, acima de todas, deveria reconhecer que artigos científicos da IEEE (o padrão de facto) são escritos tanto no Word como em LaTeX. Aliás, a imagem abaixo é de um _template_ para um _paper_ feito com gerador do site **para LaTeX** para artigos **na tua área** com as _guidelines_ do IEEE. A norma da IEEE define 10, 11 ou 12pt. O tamanho da fonte por defeito é 10pt. No entanto, como usa TeX points então não é 10 pt, mas 9.9626 pt. **Já tentei argumentar com o CC sobre outras coisas, recebendo uma resposta _seca_ ou nenhuma.** Portanto, quem te fez esta _rasteira_ é míope. Não há mais nada a dizer.

![imagem](https://user-images.githubusercontent.com/944511/151897142-d23049b6-b0b5-46bf-ba08-e19357b432c4.png)

Basicamente, o que o CC da UM está a dizer é para usar Word ou Adobe InDesign e não LaTeX.  Quem te avaliou a tese é de uma miopia gritante, quando nas áreas STEM o LaTeX é amplamente usado. Tão usado que a Springer e a IEEE têm classes em LaTeX para o efeito.

> "Não se pode discutir o oceano com um sapo do poço - ele está limitado pelo espaço em que vive. Não se pode discutir o gelo com um inseto de verão - ele está preso a uma única estação do ano".  — Zhuang Zhou

**Reply by b-pereira** [reply]
@dlrsb deixa ver o que o @joaomlourenco  te diz sobre isso. No entanto, podes tentar argumentar que a tua tese é feita em LaTeX, logo usa TeX points definido por Donald Knuth e mudar isso não é trivial. Podes mostrar que os artigos da IEEE usam TeX points mesmo quando eles definem pontos por pica tanto para um lado como para o outro. **Mas como disse, não tenhas muitas esperanças.** Na secretaria do CC não vão entender e remeter para aquilo que conhecem - o Word ou o Adobe InDesign. Falar-lhes sobre LaTeX é como lhes falar em unicórnios. A única coisa que podes tentar fazer é falar com o teu orientador e ver se ele consegue fazer com a secretaria do CC entenda o teu caso.

**Reply by b-pereira** [reply]
Para marcar o ponto, vou mostrar que o próprio documento do Despacho RT-31/2019 usar TeX points. O documento deveria ter 10pt, no entanto, o Inkscape marca 9.96. _I rest my case._

![image](https://user-images.githubusercontent.com/944511/151993956-fce897ee-21a9-42b9-8fcd-6d073aa0ce67.png)

**Comment by b-pereira** [comment]
@joaomlourenco looks like sending samples to the School of Engineering's Scientific Council was a shot in the foot.

**Comment by joaomlourenco** [comment]
Dear @b-pereira et all,
I cannot follow this very very long messages which address dozens of things in the same message.  **Please** let's try to keep the messages short (as short as possible) and direct to the point.
Also, whenever possible, please point out:
* The problem
* A possible solution or strategy ti address the identified problem.
Thank you.

**Reply by dlrsb** [reply]
@joaomlourenco , sorry for the long and confusing messages.

To summarize:
Yesterday I compiled the template (downloaded from master, uminho-defaults.ldf has the patch) without changing anything besides 1_novathesis.tex. 
- When using Adobe Acrobat Pro, Inkscape or extracting text from the PDF with the corresponding font sizes using the pdfminer package in Python, we see **12 pt in the Copyright text, and ~11.96 pt elsewhere.**
- When using PDF X-Change Editor and LibreOffice Draw, the font size is 12 pt throughout, but that's probably due to rounding.

I haven't been able to identify a possible solution, but I wonder if the patch somehow works for the Copyright section and not the rest?

**Comment by joaomlourenco** [comment]
Open `Schools/uminho/uminho-defaults.ldf` and add
```latex
\p@=1bp
```
It will do the trick.  Please confirm.

**NOTE:** it seems it does not do the trick.  It should but it does not.  I'll investigate further.

**Reply by dlrsb** [reply]
@joaomlourenco I confirm that after changing the ldf file, most of the document text is now 12pt in Adobe Acrobat Pro (Inkscape gives me 11.9999, pdfminer gives me several values that round to 12pt), except a few lines here and there like the "Statement of Integrity" title or "List of Figures" in the Contents, which are still ~11.96pt

**Reply by joaomlourenco** [reply]
These are good news.   

Initially I got the feeling it was working all right, then that it was not.  Maybe I stumbled upon one of those that remain at 11.96.

From your message can I assume it is working for normal text but not for bold (and probably also not for italic)?  Can you verify this please?  Thanks.

**Reply by dlrsb** [reply]
@joaomlourenco Yes, that seems to be the pattern. Bold and italic text, ~11.96. Normal text 12pt

**Reply by joaomlourenco** [reply]
I'll see if I can fix this. :)

**Reply by b-pereira** [reply]
~~That's a really ingenious solution! Never thought of that!~~

From Peter Flynn's article:

![image](https://user-images.githubusercontent.com/944511/151991461-ab9308af-c32d-42e1-becb-038e4f7c7c6c.png)

**Reply by b-pereira** [reply]
The definition of the point value has the problem of being used on many packages. One can change the point value at one place, but other packages can load the Tex point value definition, rendering this definition useless. As Peter Flynn pointed out, this is an internal definition. 

For instance, as you know, there are definitions for compatibility for the time of Latex 2e and the previous one coexisted in many packages. In 2005 was already regarded as unnecessary, given that LaTeX 2e was already the only version. But again, if it ain't broke, don't fix it. This part is just a reminder that LaTeX, or better TeX, is old ...

An example, to illustrate to everyone the overwrite, would be using André Miède's Classic Thesis with Ars Classica and with `scrartcl` - the three use the TeX point value. To be more specific, you did reset the point value and it works for the main text. I don't know the inner workings of `fontspec` but to apply the `FakeSlant` it might pick the default point size definition. This is embedded in TeX and I mentioned that TeX is old. Every package that derives from another or can import the TeX point definition will use it. If it wasn't for the scaling factor in `fontspec` you probably would have to redefine the point size in many places and that is not clean; and in what places, that would be the question.

**Comment by b-pereira** [comment, marked as answer]
## Answer

@dlrsb I found the answer.

It works for all the cases that you specified like the "Statement of Integrity" title or "List of Figures" in the Contents, bold and italic fonts. Check the image at the end of the answer.

Given that this template uses `\fontspec` for News Got T the way to do it is to scale all News Got T font definitions with the `Scale` option.

The scaling factor is 12/11.955 = 1.003764115 and it gives 12pt on the spot (not 11.99pt).

Go to `NOVAthesisFiles/FontStyles/FontStyles/newsgott.ldf` and add `Scale = 1.003764115` to **all** NewsGotT definitions, like below:

```latex
\setmainfont{NewsGotT}[
  Ligatures=TeX,
  Mapping=tex-text,
  Path = NOVAthesisFiles/FontStyles/Fonts/,
  UprightFont=n015003t.ttf,
  BoldFont=n015006t.ttf,
  AutoFakeSlant=0.15,
  BoldSlantedFont=n015006t.ttf,
  BoldItalicFont=n015006t.ttf,
  BoldSlantedFeatures={FakeSlant=0.15},
  BoldItalicFeatures={FakeSlant=0.15},
  Scale = 1.003764115 % ------------------------------- HERE
]

\setfontfamily\newsgott{n015003t.ttf}[
  Ligatures=TeX,
  Mapping=tex-text,
  Path = NOVAthesisFiles/FontStyles/Fonts/,
  Scale = 1.003764115 % ------------------------------- HERE
]

\setfontfamily\newsgottbf{n015006t.ttf}[
  Ligatures=TeX,
  Mapping=tex-text,
  Path = NOVAthesisFiles/FontStyles/Fonts/,
  Scale = 1.003764115 % ------------------------------- HERE
]

\setfontfamily\newsgottbfit{n015006t.ttf}[
  Ligatures=TeX,
  Mapping=tex-text,
  Path = NOVAthesisFiles/FontStyles/Fonts/,
  BoldItalicFeatures={FakeSlant=0.15},
  Scale = 1.003764115 % ------------------------------- HERE
]

\setfontfamily\newsgottit{n015003t.ttf}[
  Ligatures=TeX,
  Mapping=tex-text,
  Path = NOVAthesisFiles/FontStyles/Fonts/,
  ItalicFeatures={FakeSlant=0.15},
  Scale = 1.003764115 % ------------------------------- HERE
]

\setfontfamily\newsgottsl{n015003t.ttf}[
  Ligatures=TeX,
  Mapping=tex-text,
  Path = NOVAthesisFiles/FontStyles/Fonts/,
  AutoFakeSlant=0.15,
  Scale = 1.003764115 % ------------------------------- HERE
]
```
Also, to do this don't add @joaomlourenco  point definition because it will set the main font to 12.045 pt. ~~However, you can still play with that point definition just after the `\AtEndPreamble` line and apply the factor above to the other cases without clashing. This would be to set the math fonts point value as the main font.~~


## ~~An important note~~

~~The monospaced font used is a default, defined somewhere else in the template. Therefore, the font size for the monospaced fonts is still 11.955 pt.~~

~~In the first version of UMINHO's template, I added the monospaced font Hack (a free SIL font) and that definition is still in `newsgott.ldf` but commented. If you want, you can download that font and set `Scale` to that font. Note that the `Scale` is already there and commented with a factor of 0.8. If you want a 12pt monospaced font, set the scale to Scale = 1.003764115.~~

~~The other way would be to change the monofont font family globally and that doesn't use `fontspec`.~~


![image](https://user-images.githubusercontent.com/944511/152019868-a5937885-16f7-47f3-a156-9cf21b378440.png)

**Reply by dlrsb** [reply]
Thanks! I'll try this solution

**Reply by b-pereira** [reply]
@dlrsb I'll need your help with something. You mentioned that you customized the back cover with mentions of the financial support and logos. In https://github.com/b-pereira/UMINHO-PHD-MSc-Covers/releases I didn't create the proper box for that content yet. I f you could give me your example, it would be much appreciated.

**Reply by joaomlourenco** [reply]
@b-pereira I'm not sure if @dlrsb adaptation of the back cover is for the old UMinho branch.  If it is, it would be great to "import" it to the current main branch.  I'm available to help if necessary. Thanks.

**Reply by b-pereira** [reply]
He used the AI file from the official channel and I need to see how he did it to add to my own. The solution is already in the issue for the covers and I add that to his colleague that was using the old UMinho branch. What seemed to be me nitpicking at irrelevant details was shown to be self-evident by ... oh boy ... the 3% smaller font size.

**Reply by dlrsb** [reply]
Yes, I edited the official files using Inkscape. It should look something like [this](https://www.dropbox.com/s/xry43h4ftdnt3c3/CAPA-Tese-Doutoramento-EE_logos.pdf?dl=0)

**Reply by b-pereira** [reply]
I was studying that specification. The text (the entities names)  has to be white as the rest of the cover.  The logos have to be with the colour, even with the logos text colour. The formatting guidelines say "the entities names **or** logos". It would be better to align the image to the date and "UMinho | 2019". The guidelines only say at the bottom, but if the bottom has 10mm it would be more "correct" I think. Thank you!

**Comment by joaomlourenco** [comment]
@b-pereira and @dlrsb 

There was a problem with the main font.  @b-pereira suggestion of changing the scale is, indeed, the perfect solution!  Congrats on the idea.  Please submit a patched `newsgott.ldf`.  Thanks.

Now, concerning the mono fonts, the inter-line spacing, etc…  my advice is “**don't create problems where there is none!**”  Submit again and let _them_ point out (or not) other problems.

**Reply by b-pereira** [reply]
@joaomlourenco Pull request created https://github.com/joaomlourenco/novathesis/pull/290
